.upptimerc.yml

```diff
@@ -48,6 +48,10 @@ sites:
     url: https://charter.africa
     icon: https://charter.africa/images/charter-logo.svg
 
+  - name: charterAFRICA - Media
+    url: https://charterafrica.prod.codeforafrica.org/media/charter-logo.svg
+    icon: https://charterafrica.prod.codeforafrica.org/media/charter-logo.svg
+    
```

Comment:
1. Does this file come from S3 or is it the ones we have in code i.e. `public` folder?
2. Why are we not use _charter.africa_ domain i.e. `https://charter.africa/media/charter-logo.svg`?

Reply:
@kilemensi

1. Does this file come from S3 or is it the ones we have in code i.e. public folder?

This should be served from S3, just replicated on `charterafrica-dev`, disabling S3 returns 404

2.Why are we not use charter.africa domain i.e. https://charter.africa/media/charter-logo.svg?

The actual served request URL is `charterafrica.prod`, though the `charter.africa` resolves to the same, unless there's a preference for the later

Reply:
Actually all media files are served from S3

Reply:
... yeah, weird seeing codeforafrica.org in the URL there since this site is for a separate org.